powerauth-push-server/src/main/resources/application.properties

```diff
@@ -56,7 +56,7 @@ powerauth.push.service.registration.retry.maxAttempts=2
 powerauth.push.service.clients.cache.refreshAfterWrite=5m
 
 # APNs Configuration
-powerauth.push.service.apns.useDevelopment=true
+powerauth.push.service.apns.useDevelopment=false
```

Comment:
👀 Shouldn't we document this change?

Reply:
Yes, I will prepare documentation.

Reply:
I will revert this change, because it complicates development. The environment should be chosen automatically during registration.